master/internal/trial.go

```diff
@@ -386,6 +386,13 @@ func (t *trial) maybeAllocateTask() error {
 
 	isSingleNode := t.config.Resources().IsSingleNode() != nil && *t.config.Resources().IsSingleNode()
 
+	// This should be set to defaults from TaskContainerDefaults, but catch case where we're
+	// restarting a trial previous to this field being set.
+	preemptionTimeout := model.DefaultPreemptionTimeout
+	if t.config.PreemptionTimeout() != nil {
```

Comment:
ah, ok; this is from `InternalConfig` which uses a pointer for duration so it can be null